### PR TITLE
Chocobo Circuit zone update from caps

### PR DESCRIPTION
**Pull request by Tom-Neverwinter**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

adds the sql database entries missing from zone 70 Chocobo Circuit from cap files

## Steps to test these changes

run around the zone and confirm npcs are updated

### CODE REVIEW

File: sql/npc_list.sql [hunk 1]
```diff
@@ -6739,213 +6739,214 @@ INSERT INTO `npc_list` VALUES (17060176,'blank','     ',0,0.000,0.000,0.000,1,50
...
-INSERT INTO `npc_list` VALUES (17064120,'Sigiswald','Sigiswald',199,-112.255,-11.500,-103.019,3260,15,50,0,0,0,0,27,0x00005A0000000000000000000000000000000000,32,'TOAU',1);
-INSERT INTO `npc_list` VALUES (17064121,'Rhind_Mhikkrol','Rhind Mhikkrol',199,-11.520,-11.500,-102.751,1109,44,50,0,0,0,0,27,0x0100000700100420113003401150006000700000,32,'TOAU',1);
-INSERT INTO `npc_list` VALUES (17064122,'Makrena-Rukrena','Makrena-Rukrena',199,-17.958,-13.000,-115.173,673,44,50,0,0,0,0,27,0x0100000577100720003017401250006000700000,32,'TOAU',1);
-INSERT INTO `npc_list` VALUES (17064123,'Denpepe','Denpepe',199,-11.523,-14.500,-125.579,27,44,50,0,0,0,2,27,0x0100000600102C20953003401350006000700000,32,'TOAU',1);
-INSERT INTO `npc_list` VALUES (17064124,'Maik','Maik',193,-107.098,-11.500,-102.743,2560,18,50,0,0,0,0,27,0x0000640000000000000000000000000000000000,32,NULL,1);
+INSERT INTO `npc_list` VALUES (17064120,'Sigiswald','Sigiswald',199,-112.255,-11.500,-103.020,27,15,50,0,0,0,0,27,0x00005A0000000000000000000000000000000000,32,'TAOU',1);
```
Comment (WinterSolstice8) [inline sql/npc_list.sql:6915]: find/replace
`TAOU` -> `TOAU`

**Review by WinterSolstice8 — CHANGES_REQUESTED** [review]

File: sql/npc_list.sql (hunk 1 above)
Comment (Tom-Neverwinter) [reply sql/npc_list.sql:6915]: Should be good, fixed another mistake I made as well

**Review by WinterSolstice8 — APPROVED** [review]
lgtm

File: sql/npc_list.sql [hunk 2]
```diff
@@ -6739,213 +6739,214 @@ INSERT INTO `npc_list` VALUES (17060176,'blank','     ',0,0.000,0.000,0.000,1,50
...
+INSERT INTO `npc_list` VALUES (17064133,'DealerMoogle','Dealer Moogle',131,-307.559,0.000,-480.407,14,50,50,0,0,0,0,3,0x0000D50300000000000000000000000000000000,0,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17064134,'Gate:ChocoboC','Gate: Chocobo Circuit',0,-482.700,-1.250,-371.000,32769,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17064135,'Gate:ChocoboC','Gate: Chocobo Circuit',0,-514.159,-1.350,-524.358,32769,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17064136,'Gate:ChocoboC','Gate: Chocobo Circuit',0,-136.000,-1.499,-521.750,32769,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17064137,'Gate:ChocoboC','Gate: Chocobo Circuit',0,-340.000,-1.500,-314.299,32769,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17064138,'Gate:ChocoboC','Gate: Chocobo Circuit',0,-150.000,-2.000,-392.750,32769,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
```
Comment (Xaver-DaRed) [inline sql/npc_list.sql:6933]: `Gate:ChocoboC` is not a valid file name. Please chage it to `GateChocoboC`

File: sql/npc_list.sql [hunk 3]
```diff
@@ -24789,28 +24790,28 @@ INSERT INTO `npc_list` VALUES (17666804,'Moogle','Moogle',0,0.000,0.000,0.000,0,
...
+INSERT INTO `npc_list` VALUES (17670606,'qm_orthus_2','???',0,-400.000,0.900,97.000,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (17670607,'qm_dragua_2','???',0,-221.000,0.800,-350.000,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (17670608,'qm_bennu_2','???',0,102.000,0.000,-151.000,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (17670609,'qm_rani_3','???',0,-823.000,-8.400,-390.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (17670610,'qm_orthus_3','???',0,-400.000,0.150,127.000,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (17670611,'qm_dragua_3','???',0,-221.000,0.950,-320.000,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (17670612,'qm_bennu_3','???',0,80.000,0.250,-129.000,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
```
Comment (Xaver-DaRed) [inline sql/npc_list.sql]: This changes (Abyssea Altepa) shouldnt be here

Comment (Tom-Neverwinter) [reply sql/npc_list.sql]: Fetch and merge resolves this. [maybe not] 

``I'll have to redownload npc list and manually edit it again then copy pasta zone 70``

https://raw.githubusercontent.com/LandSandBoat/server/base/sql/npc_list.sql

This insnt a change from my code. viewing the raw of the base its in there too. I only care about zone 70 in this one.

File: sql/npc_list.sql (hunk 2 above)
Comment (Tom-Neverwinter) [reply sql/npc_list.sql:6933]: Completed as requested

**Review by Xaver-DaRed — DISMISSED** [review]
EDIT: Change what Claywar pointed out and this should be good to go.

File: sql/npc_list.sql [hunk 4]
```diff
@@ -6739,213 +6739,214 @@ INSERT INTO `npc_list` VALUES (17060176,'blank','     ',0,0.000,0.000,0.000,1,50
...
+INSERT INTO `npc_list` VALUES (17064149,'GreeterMoogle','Greeter Moogle',0,-332.000,0.000,-480.000,14,50,50,0,0,0,2,3,0x0000730900000000000000000000000000000000,0,'SOA',1);
 INSERT INTO `npc_list` VALUES (17064150,'Achieve_Master','Achieve Master',0,0.000,0.000,0.000,0,50,50,0,0,0,2,3,0x0000320000000000000000000000000000000000,0,'SOA',0);
-INSERT INTO `npc_list` VALUES (17064151,'Unity_Master','Unity Master',0,0.000,0.000,0.000,0,50,50,0,0,96,2,2051,0x0000340000000000000000000000000000000000,0,'SOA',0);
+INSERT INTO `npc_list` VALUES (17064151,'Unity_Master','Unity Master',0,0.000,0.000,0.000,0,50,50,0,0,96,2,2075,0x0000730900000000000000000000000000000000,0,'SOA',0);
+INSERT INTO `npc_list` VALUES (17064152,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,112,2,2075,0x00003D0900000000000000000000000000000000,32,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17064153,'MandragoraAssi','Mandragora Assistant',192,-320.000,-0.850,-493.200,32782,50,50,0,0,32,0,1,0x00008F0B00000000000000000000000000000000,2,'NULL',1);
```
Comment (claywar) [inline sql/npc_list.sql:6948]: This is why CI is failing. Null Should not be in quotes

**Review by claywar — APPROVED** [review]

**Pull request merged**